src/Area/Area.scss

```diff
@@ -0,0 +1,37 @@
+.area-card {
+  height: 300px;
```

Comment:
If we could shift these into percentages eventually that would probably be more ideal for scaling, with some of the initial styling rules I set up a height and width of 100% are passed down from the top elements, so there should always be a relative container above to reference. 

That said an argument could probably be made for a fixed width if things were scaling strangely, if we went that route we'd just want to change things elsewhere.